### ISSUE TITLE
How to Train porn Class Only?

### ISSUE THREAD

**Issue by chenzi** [GantMan/nsfw_model#15]
I have 10,000 pornographic pictures.   How to Train porn Class Only?  I trained only one category to predict that all pictures were pornographic.

**Comment by GantMan** [comment]
You'll want to make a porn/not_porn group of roughly the same size.

It's 2 categories.  Preferably have the not_porn folder have a lot of images of people interacting, but not pornographically.

**Comment by chenzi** [comment]
How do I train a class based on the model [https://s3.amazonaws.com/nsfwdetector/nsfw.299x299.h5](url)， I want to train my 10000  pornographic pictures to porn class.

**Comment by chenzi** [comment]
but I have not other class pictures

**Comment by GantMan** [comment]
I think you need to get 10,000 non-porn photos.  Then use the training code you have, but change the final layer to softmax sigmoid with 2 neurons.   Porn and not-porn.

**Comment by GantMan** [comment]
ACTUALLY, you could use "K-Nearest Neighbors Classifier" to turn the model into classifying 2 different items.  If you use the inference data.  I'm not an expert on how to do this, though.
